XLS-0070d-credentials/README.md

```diff
@@ -74,18 +74,21 @@ This ledger object is an on-chain representation of a credential.
 
 This object costs one owner reserve for either the issuer or the subject of the credential, depending on whether the subject has accepted it.
 
+The `Credential` object will live in both the `Subject` and `Issuer`'s owner directories (similar to a trustline or escrow).
+
 ### 2.1. Fields
 
 | Field Name | Required? | JSON Type | Internal Type | Description |
 |------------|-----------|-----------|---------------|-------------|
-|`LedgerEntryType`| ✔️|`string`|`UInt16`|The ledger object's type (`Credential`).|
-|`Flags`| ✔️|`number`|`UInt32`|Flag values associated with this object.|
-|`Subject`| ✔️|`string`|`AccountID`|The account that the credential is for.|
-|`Issuer`| ✔️|`string`|`AccountID`|The issuer of the credential.|
-|`CredentialType`| ✔️|`string`|`Blob`|A (hex-encoded) value to identify the type of credential from the issuer.|
+|`LedgerEntryType`|✔️|`string`|`UInt16`|The ledger object's type (`Credential`).|
+|`Flags`|✔️|`number`|`UInt32`|Flag values associated with this object.|
```

Comment:
Re: value of the flag: |`lsfAccepted`|`0x00010000`|, why was this particular value chosen?
Visually, `0x00000001` (or) `0x10000000` would have been easier on the eyes (users could have set first or the last bit)

Why was `Flags` allocated 32 bits? The current functionality could have been accomplished with a smaller type (e.g. `UINT8`). This was the smallest type that I could think of in `rippled`, are there any one-bit-sized types?

Reply:
The least important two bytes are [reserved](https://xrpl.org/docs/references/protocol/transactions/common-fields#flag-ranges). So this is the smallest allowed value.

The field `Flags` is a [common field included in all transactions](https://xrpl.org/docs/references/protocol/transactions/common-fields). I can't change its size.